Documentation/Signoffs/jiegwuatu42-web_wireless-signoff.md

```diff
@@ -7,22 +7,36 @@ The Wireless acquisition of data helps effortlessly boost the quality of life. T
 
 The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner. Constraints on the subsystem are as follows:
 1. The server host machine must possess adequate storage space for the server code to store all collected data.
-2. The data will be encrypted with an undisclosed algorithm
+2. The data will be encrypted and decrypted with an undisclosed algorithm using AES Symmetric Encryption.  
+3. The server must be able to be established and connected to the internet.
+4. The server must have adequate storage for user data.
+5. The data received must be in legible format.
 
 
 ## BUILDABLE SCHEMATIC OF SUBSYSTEM
 
+<img width="350" alt="4G Connection" src="https://user-images.githubusercontent.com/110966922/219462337-5e72409b-17d1-4c59-8d68-eab86c8fda1f.png">
+<img width="432" alt="Wifi Connection" src="https://user-images.githubusercontent.com/110966922/219462340-b16763c0-cbe4-4803-ba0f-be77fbb212e4.png">
 
 
+Server will be made using HTML. It will receive info from TCP Connection provided from UART. 
 
 ## ANALYSIS OF CHOSEN COMPONENT(S)
 
-
+The team has chosen HTML5 UP to statisfy constraints and jobs of this system for the server/UI:
+1. Free web design templates that keeps price of the project affordable.
+2. Fully responsive software that allows easy between web and mobile platforms/
 
 ## BILL OF MATERIALS (BOM)
 
-| Designator | Manufacturer | Manufactured Part # | Description                                                   | Quantity | Price Each  |
-| ---------- | ------------ | ------------------- | ------------------------------------------------------------- | -------- | ----------  |
-|            |              |                     |                                                               |          |             |
-|            |              |                     |                                                               |          |             |
+N/A
+
+## References
+1. A. Marget, “Data Encryption: How It Works & Methods Used,” Unitrends, Jan. 27, 2022. https://www.unitrends.com/blog/data-encryption#:~:text=A%20key%20is%20used%20to (accessed Feb. 10, 2023).
```

Comment:
None of these are references above. Most (all?) of these are superficial.

Reply:
Review the reference list after the previous comments regarding system components are resolved and remove the irrelevant ones.

Reply:
Review the references list....